packages/react/src/SubdomainNavBar/SubdomainNavBar.module.css

```diff
@@ -593,7 +593,15 @@
     var(--brand-SubdomainNavBar-search-results-border-default, #b7bfc7);
 }
 
-.SubdomainNavBar-search-result-item-heading a {
+.SubdomainNavBar-search-result-item-container {
+  font-weight: var(--brand-heading-weight-400);
```

Comment:
Could these be applied to the links directly instead of the container?

Reply:
I reduced the amount of styles that lived within the container. It seems that `letter-spacing` and `font-size` need to be applied within the container to have a 1:1 match with the current styles where the `h6` is the container. 

I'm assuming that these styles apply to the container itself, as whenever I applied them to the links directly there was a small style difference between the current style of the list items in prod, compared to the ones within this PR 🤔